src/client/components/pages/entities/title.js

```diff
@@ -34,17 +34,18 @@ function EntityTitle({entity, handleModalToggle}) {
 	const label = getEntityLabel(entity);
 	return (
 		<div>
-			<h1>
+			<h1 className="d-flex justify-content-between">
```

Comment:
Not so sure about this, it seems to cause an issue with disambiguations:
![image](https://github.com/user-attachments/assets/b79035f1-b375-4877-98e6-db304dd9289e)

In a flex container, you can add `margin-left: auto` to the last element (the review button) to automatically align to the right. Maybe that can work here instead of justify-content-between on the parent div?